Modix.Services/Utilities/EmojiUtilities.cs

```diff
@@ -62,6 +62,6 @@ static bool IsVariantSelector(Rune rune)
 
-        private static readonly Regex _builtInEmojiRegex = new Regex(EmojiPattern, RegexOptions.Compiled);
+        private static readonly Regex _builtInEmojiRegex = new(EmojiPattern, RegexOptions.Compiled);
```

Comment:
How evil is this, and can we use .net 7's regex source generator?

Reply:
Yeah, that's definitely on my todo list. Been having a little trouble trying to upgrade to .NET 7 at the moment, but I do want to make some improvements here